internal/transactionsubmission/services/horizon_test.go

```diff
@@ -404,8 +439,6 @@ func Test_DeleteChannelAccountOnChain(t *testing.T) {
 		})
 	}
 
-	mockSigService.AssertExpectations(t)
 	horizonClientMock.AssertExpectations(t)
 	privateKeyEncrypterMock.AssertExpectations(t)
-	mLedgerNumberTracker.AssertExpectations(t)
```

Comment:
do you need to add assertions for `mHostSigner`?

Reply:
Good point! 

I found out that the `mockery` lib already asserts the expectations automatically when you create the mocks with its constructor, like in:

https://github.com/stellar/stellar-disbursement-platform-backend/blob/6855158841998df80bbfcb1a5965fbe3d3c75fd3/internal/transactionsubmission/engine/mocks/ledger_number_tracker.go#L70-L78